meetings/12-06-18.md

```diff
@@ -0,0 +1,203 @@
+#Node.js Promise Post Mortem Debugging Meeting 12-06-2018
```

Comment:
`s/#Node/# Node/` otherwise GH rendering will fail:

![image](https://user-images.githubusercontent.com/4048656/41316778-21c6860a-6e48-11e8-854a-ee5134cdf83b.png)